app/controllers/api/v3/courses_controller.rb

```diff
@@ -4,6 +4,7 @@ class CoursesController < API::V3::ApplicationController
       before_action :build_recruitment_cycle
       before_action :build_provider
       before_action :build_courses
+      before_action :add_findable_filter
```

Comment:
only applicable to `index`
```suggestion
      before_action :add_findable_filter, only: [:index]
```